scheme/psa-iot/psa_extractor.go

```diff
@@ -0,0 +1,158 @@
+// Copyright 2024 Contributors to the Veraison project.
+// SPDX-License-Identifier: Apache-2.0
+package psa_iot
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"reflect"
+
+	"github.com/veraison/corim/comid"
+	"github.com/veraison/services/handler"
+	"github.com/veraison/services/scheme/common/arm/platform"
+)
+
+type PsaExtractor struct {
+	Profile string
+}
+
+func (o PsaExtractor) RefValExtractor(rv comid.ReferenceValue) ([]*handler.Endorsement, error) {
+	var classAttrs platform.ClassAttributes
+
+	if (o.Profile == "") || (o.Profile != "http://arm.com/psa/iot/1") {
```

Comment:
Can just be `if o.Profile != "http://arm.com/psa/iot/1"`

Reply:
Make sense, will do the correction!

Reply:
Done!